evml/keras/models.py

```diff
@@ -226,7 +231,17 @@ def calc_uncertainties(self, preds, y_scaler):
             aleatoric[:, i] *= self.training_var[i]
             epistemic[:, i] *= self.training_var[i]
 
-        return mu, np.sqrt(aleatoric), np.sqrt(epistemic)
+        return mu, aleatoric, epistemic
+    
+    def predict_dist_params(self, x):
+        preds = self.model.predict(x, batch_size=self.batch_size)
+        mu, v, alpha, beta = np.split(preds, 4, axis=-1)
+        if mu.shape[-1] == 1:
+            mu = np.expand_dims(mu, 1)
+        if y_scaler:
```

Comment:
There is no y_scaler input to this function, resulting in the flake8 failure. Can you fix?

Reply:
Thanks; fixed!